composer.json

```diff
@@ -15,7 +15,8 @@
     "autoload": {
         "psr-4": {
             "Maleficarum\\Response\\Http\\": "src/Response/Http/",
-            "Maleficarum\\Response\\Initializer\\": "src/Response/Initializer/"
+            "Maleficarum\\Response\\Initializer\\": "src/Response/Initializer/",
+            "Maleficarum\\Response\\Plugin\\": "src/Response/Plugin/"
```

Comment:
Did you find out why we list all the directories like this instead of just adding the whole `src/Response`?